feature-libs/product-configurator/rulebased/components/form/configurator-form.component.ts

```diff
@@ -100,4 +105,8 @@ export class ConfiguratorFormComponent implements OnInit {
   createGroupId(groupId?: string): string | undefined {
     return this.configUtils.createGroupId(groupId);
   }
+
+  get expMode(): Observable<boolean> {
+    return this.configExpertModeService.getExpMode();
+  }
```

Comment:
seem like that there was no unit test added for this condition

Reply:
Covered indirectly.

Reply:
Covered indirectly does not mean tested, or?

Reply:
Done!